cayleypy/graphs_lib.py

```diff
@@ -46,6 +46,8 @@ def prepare_graph(name, n=0) -> CayleyGraph:
       * "all_transpositions" - Cayley graph for S_n (n>=2), generated by all n(n-1)/2 transpositions.
       * "pancake" - Cayley graph for S_n (n>=2), generated by reverses of all prefixes. It has n-1 generators denoted
           R1,R2..R(n-1), where Ri is reverse of elements 0,1..i. See https://en.wikipedia.org/wiki/Pancake_graph.
+      * "burnt_pancake" - Cayley graph generated by reverses of all signed prefixes. Actually is a graph for 
+         S_2n (n>=1) and has n generators denoted R1,R2..R(n), where Ri is reverse of elements 0,1..i,n,n+1..n+i. 
```

Comment:
Ri is composition of reversing elements `0,1..i` and reversing elements `n,n+1..n+i`.